src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -136,16 +136,20 @@ public string Accept
 
         public virtual bool AllowReadStreamBuffering
         {
+            // Same behavior as .NET Framework.
```

Comment:
I don't find the comment obvious in that case. Having a test that runs in all platforms and ensures the right exception would be preferable to a comment. (I thought I saw a test change already.)